lib/internal/fs/utils.js

```diff
@@ -727,6 +729,24 @@ const validateBufferArray = hideStackFrames((buffers, propName = 'buffers') => {
   return buffers;
 });
 
+const validateAndNormalizeBufferArray = hideStackFrames((buffers, propName = 'buffers') => {
+  if (!ArrayIsArray(buffers))
+    throw new ERR_INVALID_ARG_TYPE(propName, 'ArrayBufferView[]', buffers);
```

Comment:
The error message should reflect that `ArrayBuffer` are also accepted. How about this?
```suggestion
    throw new ERR_INVALID_ARG_TYPE(propName, ['ArrayBufferView[]', 'ArrayBuffer[]'], buffers);
```
Same thing for [`validateStringAfterArrayBufferView`](https://github.com/nodejs/node/blob/a37c083dc138bb648be9a3581d1b2fe3462d4c8b/lib/internal/fs/utils.js#L890) and [`validateBufferArray`](https://github.com/nodejs/node/blob/a37c083dc138bb648be9a3581d1b2fe3462d4c8b/lib/internal/fs/utils.js#L720)